memory_windows.go

```diff
@@ -21,12 +21,49 @@ type win32OperatingSystem struct {
 	TotalVisibleMemorySize uint64
 }
 
+const wmqlPhysicalMemor = "SELECT BankLabel, Capacity, DataWidth, Description, DeviceLocator, Manufacturer, Model, Name, PartNumber, PositionInRow, SerialNumber, Speed, Tag, TotalWidth FROM Win32_PhysicalMemory"
+
+type win32PhysicalMemory struct {
+	BankLabel     string
+	Capacity      uint64
+	DataWidth     uint16
+	Description   string
+	DeviceLocator string
+	Manufacturer  string
+	Model         string
+	Name          string
+	PartNumber    string
+	PositionInRow uint32
+	SerialNumber  string
+	Speed         uint32
```

Comment:
In a future PR (not this one), I'd be interested in adding a `Speed` field to the `MemoryModule` struct. Need to have a conversation about what the appropriate data type is, though, so let's leave that for a future commit.